release-blog/2024/2024-10-23.0.mdx

```diff
@@ -0,0 +1,119 @@
+with the `covering` metadata included. So you can now do this:
+```
+ogr2ogr --config AWS_NO_SIGN_REQUEST=yes -spat -71.068 42.353 -71.058 42.363  -f GeoJSON output.geojson
+/vsis3/overturemaps-us-west-2/release/2024-10-23.0/theme=buildings/type=building/
+```
+:::
+
+You can access the datasets by following the process outlined [here](https://docs.overturemaps.org/getting-data/). We encourage users to report bugs and provide feedback on the data, schema, and GERS on one of our Overture GitHub repositories:
+
+- [Schema](https://github.com/OvertureMaps/schema)
+- [Data](https://github.com/OvertureMaps/data)
+- [Docs](https://github.com/OvertureMaps/docs)
+- [Python command-line tool](https://github.com/OvertureMaps/overturemaps-py)
+- [Explore site](https://github.com/OvertureMaps/io-site)
+
+
+## Breaking changes
+
+We removed the `connector_ids` property from the schema. It has been replaced by the new `connectors` property. You'll find more information [here](https://docs.overturemaps.org/blog/2024/08/15/preview-august-release/#connectors).
+
+## Deprecations
+
+In the `2024-08-20.0` [release notes](https://docs.overturemaps.org/release/2024-08-20.0/), we announced the deprecation of the `connector_ids` property in the transportation schema. We have removed that property in this release. 
+It has been replaced by the new `connectors` property.
+
+## Theme-specific updates
+
+:::info
+The base, buildings, divisions, and places themes are in GA. The transportation theme is in beta, and the addresses theme is in alpha.
+:::
+
+### Addresses
+- added new address features in the following countries:
+    - **89.9 million** in Brazil
+    - **1.7 million** in Croatia
+    - **5.9 million** in German[y](https://youtu.be/WBpDs-BfwO0?feature=shared)
+    - **19.6 million** in Japan
+    - **2.6 million** in Norway
+    - **7.7 million** in Poland
+    - **1.6 million** in Slovakia
+- updated the [US National Address Database](https://www.transportation.gov/gis/national-address-database) to v17
+- added data from the [NYC Open Data](https://opendata.cityofnewyork.us/) portal
+- fixed a bug that resulted in both the US National Address Database and NYC Open Data being included for Queens
+- fixed a typo in the Geoscape Australia source name
+- changed the data source name for Chile to CL-INE and Portugal to PT-INE to distinguish the two
+
+### Base
+- fixed lake features in Manitoba, Canada
+
+### Buildings
+- fixed building features with class but no subtype · Issue #183 · OvertureMaps/data 
+- fixed height data missing in September release 
+- fixed building height but there is no source information 
+- fixed building heights derived from lidar in areas with tree cover are incorrect 
+- fixed ML building in water 
+- fixed lidar heights incorrectly being added 
+- added new data from OSM (2024-10-05)
+- added new data from Microsoft (2024-09-24)
+- added new from Esri (2024-10-15)
+
+### Divisions
+- changed subtype of Puerto Rico to dependency
```

Comment:
This should change to "updated the subtype of 38 dependencies" or similar. The theme maintained these places as countries, but in this release, they're (more correctly) assigned a dependency subtype.